src/main/java/uk/co/ramp/covid/simulation/place/Household.java

```diff
@@ -186,23 +204,33 @@ public void doTesting(Time t) {
         }
     }
 
-    private void moveNeighbour(boolean lockdown) {
+    private void moveNeighbour(Time t, boolean lockdown) {
+        // We only visit neighbours between 9 - 20
+        if (!visitsNeighbourToday || t.getHour() < 8 || t.getHour() >= 19) {
+            return;
+        }
 
         List<Person> left = new ArrayList<>();
-        if((!lockdown) || (!this.lockCompliant)) {
+        if(!lockdown || !lockCompliant) {
 	        for (Household n : getNeighbours()) {
 	            if (n.isIsolating()) {
 	                continue;
 	            }
 
-                if (PopulationParameters.get().householdProperties.pHouseholdVisitsNeighbour.sample()) {
+	            // Since we determine if we should try to visit a neighbour a the star of the day, we have
+                // equal oppertunity of visiting a neighbour each hour they are open
+                if (new Probability(1.0 / (18 - 8)).sample()) {
```

Comment:
I am not entirely sure I understand this method, but surely the neighbours are open forom 8 - 20?